Modules/Campaign/Http/Services/CampaignService.php

```diff
@@ -1,152 +1,304 @@
-    {

-

-        $allCampaignsCount = Campaign::where('user_id', $requestData->user_id)->count();

-        $draftCampaignsCount = Campaign::where('user_id', $requestData->user_id)->where('status', 'draft')->count();

-        $scheduledCampaignsCount = Campaign::where('user_id', $requestData->user_id)->where('status', 'schedule')->count();

-        $completedCampaignsCount = Campaign::where('user_id', $requestData->user_id)->where('status', 'completed')->count();

-        $campaigns = Campaign::where('user_id', $requestData->user_id);

-        $status = strtolower($requestData->status);

-        if ($status !== 'all') {

-            $campaigns->where('status', $status);

-        }

-        $paginatedCampaigns = $campaigns->paginate(10);

-        $filteredCampaigns = [];

-        if ($paginatedCampaigns) {

-            foreach ($paginatedCampaigns as $campaign) {

-                $filteredCampaigns[] = array(

-                    'title' => $campaign->title,

-                    'campaign_key' => $campaign->campaign_key,

-                    'updated_at' => date("F j, Y, g:i a", strtotime($campaign->updated_at)),

-                );

-            }

-        }

-        $data = array(

-            "campaigns" => $filteredCampaigns,

-            "allCampaignsCount" => $allCampaignsCount,

-            "draftCampaignsCount" => $draftCampaignsCount,

-            "scheduledCampaignsCount" => $scheduledCampaignsCount,

-            "completedCampaignsCount" => $completedCampaignsCount,

-        );

-

-        return ['res' => true, 'msg' => "", 'data' => $data];

-    }

-

-    /**

-     * Get the specified campaign

-     *

-     * @param string $campaignKey

-     * @return array

-     */

-    public function get(string $campaignKey): array

-    {

-

-        $campaign = Campaign::where('campaign_key', $campaignKey)->first();

-

-        // return error if no campaign found

-        if (!$campaign) {

-            return [

-                'res' => false,

-                'msg' => 'Campaign not found !',

-                'data' => ""

-            ];

-        }

-

-        return [

-            'res' => true,

-            'msg' => '',

-            'data' => $campaign

-        ];

-    }

-

-    /**

-     * Remove the specified campaign from storage.

-     *

-     * @param string $campaignKey

-     * @return array

-     */

-    public function delete(string $campaignKey): array

-    {

-        $campaign = Campaign::where('campaign_key', $campaignKey)->first();

-

-        // return error if no campaign found

-        if (!$campaign) {

-            return [

-                'res' => false,

-                'msg' => 'Campaign not found !',

-                'data' => ""

-            ];

-        }

-

-        $campaign->delete();

-

-        return [

-            'res' => true,

-            'msg' => 'Campaign successfully deleted',

-            'data' => ""

-        ];

-    }

-

-}

+<?php
```

Comment:
@arijit1111 There is no usage of CampaignService, can you please avoid to include any not-required file in PR? You may need to delete it.